docs/rules.md

```diff
@@ -283,9 +283,54 @@ when:
     - <condition2>
 ```
 
+#### Chaining Condition Variables
+
+You can also chain the variables from one condition to be used as input in another condition in a _or_ and _and_ block of conditions
+
+Example:
+
+```yaml 
+when:
+ or:
+  - builtin.xml:
+      xpath: "//dependencies/dependency"
+      filepaths: "{{poms.filepaths}}"
```

Comment:
@shawn-hurley how can the user know what variables are available to get here? In this case it's `filepaths`, but what about the case of java conditions for instance?

Reply:
This should be published by the provider int he capabilities for a given condition. 

With that said, I don't think we have a good way of exposing this data at the moment.